components/ui-tray.vue

```diff
@@ -0,0 +1,83 @@
+<template>
```

Comment:
1) panel should be updated to this style i think, but most logical to not do this in this branch.
2) need to set a scroll on the body, see about section

Reply:
Done.